src/attendees/attendee.py

```diff
@@ -79,7 +89,17 @@ def set_attribute(self, key: Hashable, value: Any,
             TypeError if key is not hashable.
             ValueError if key is "fname", "lname", or "email"
         """
+        self.key = key
+        self.value = value
+        self.verifyFunc = verifyFunc
```

Comment:
The `Attendee` objects should not have `key`, `value`, nor `verifyFunc` attributes. They should check that `key` is not equal to `"fname"`, `"lname"`, nor `"email"` (raise a `ValueError` if they are). If `verifyFunc` is not `None`, use it to check that `value` is valid. If valid, set the attribute `key` to `value`.